test/controllers/reports_controller_test.rb

```diff
@@ -30,13 +30,13 @@ def from_alegre(pm)
     create_report_design_annotation_type
     authenticate_with_token @a
     create_dynamic_annotation annotation_type: 'report_design', set_fields: { state: 'published', options: { language: 'en', image: '' } }.to_json, action: 'save', annotated: @pm
-    pm = create_project_media team: @t, archived: 1
-    pm2 = create_project_media team: @t, quote: random_string, media: nil
-    pm3 = create_project_media team: @t
+    pm = create_project_media team: @t, media: create_valid_media, archived: 1
```

Comment:
@melsawy, I only have one question:
If we have `options[:media] = create_valid_media({team: options[:team]}) if enforce_create_media && !options.has_key?(:set_original_claim)` inside `create_project_media`, why do you need to call `create_valid_media` here?